src/lib/util.ts

```diff
@@ -268,12 +299,12 @@ export async function getNetworkName(id?: string): Promise<string> {
     case "main":
     case "1":
       return "main";
-    case "morden":
-    case "2":
-      return "morden";
-    case "ropsten":
-    case "3":
-      return "ropsten";
+    // case "morden":
```

Comment:
where is "kovan" ?

Reply:
in the code, line 315

Reply:
why there is rinkeby support in this function and not kovan ?

Reply:
the support for kovan is there, please check the code

Reply:
at the getNetworkName ?

Reply:
Here: https://github.com/daostack/alchemy/blob/a84a2ae762e869cc3a107644f3e5bd35f9a16335/src/lib/util.ts#L314